decidim-initiatives/app/models/decidim/initiative.rb

```diff
@@ -53,7 +53,7 @@ class Initiative < ApplicationRecord
              dependent: :destroy,
              as: :participatory_space
 
-    enum signature_type: [:online, :offline, :any]
+    enum signature_type: [:online, :offline, :any], _suffix: true
```

Comment:
Hello @oriolgual 

This line is causing some bugs like : 
```
ActionView::Template::Error (undefined method `offline?' for #<Decidim::Initiative:0x0000562268561cc0>
```

because of the new suffix on these methods.

If the suffix is needed, we should add : 
```ruby
    alias_method :online?, :online_signature_type?
    alias_method :offline?, :offline_signature_type?
    alias_method :any?, :any_signature_type?
```
(or fix all the template but ...) 

Anyway, since this information has been moved to `InitiativesType`, we could also remove it from `Initiative` and delegates it to the types.
```ruby
delegate :online_signature_type?, :offline_signature_type?, :any_signature_type?, to: :type
```